resources/js/views/settings/Application.vue

```diff
@@ -0,0 +1,471 @@
+        <div v-html="fieldError('favicon_file')"></div>
+      </template>
+    </b-form-group>
+
+      <!--Logo Settings-->
+      <b-form-group
+        label-for="application-logo-input"
+        :state='(fieldState("logo") === null && fieldState("logo_file") === null) ? null : false'
+      >
+
+        <template v-slot:label>
+          {{ $t('settings.application.logo.title') }}
+        </template>
+
+        <b-row class="my-3" align-v="center">
+          <b-col sm="6" lg="3" class="text-center">
+            <b-img
+              v-if="uploadLogoFileSrc!==null || settings.logo!==null"
+              :src="uploadLogoFileSrc ? uploadLogoFileSrc : settings.logo"
+              class="my-2"
+              rounded="0"
+              alt="application-logo-preview"
+              width="150"
+              height="100"
+              fluid
+            >
+            </b-img>
+          </b-col>
+          <b-col sm="6" lg="9">
+            <b-form-text v-if="!uploadLogoFile">{{ $t('settings.application.logo.urlTitle') }}</b-form-text>
+            <b-input-group>
+              <b-form-input
+                id="application-logo-input"
+                v-if="!uploadLogoFile"
+                :placeholder="$t('settings.application.logo.hint')"
+                v-model="settings.logo"
+                :disabled="isBusy || viewOnly"
+                class="my-2"
+                :state='fieldState("logo")'
+              >
+              </b-form-input>
+            </b-input-group>
+            <b-form-text v-if="!viewOnly">{{ $t('settings.application.logo.uploadTitle') }}</b-form-text>
+            <b-input-group v-if="!viewOnly">
+              <b-form-file
+                id="application-logo-form-file"
+                :state='fieldState("logo_file")'
+                :disabled="isBusy"
+                :browse-text="$t('app.browse')"
+                :placeholder="$t('settings.application.logo.selectFile')"
+                v-model="uploadLogoFile"
+                accept="image/jpeg, image/png, image/gif, image/svg+xml"
+              >
+              </b-form-file>
+              <template #append v-if="uploadLogoFile">
+                <b-button id="application-upload-button" variant="danger" @click="uploadLogoFile = null">
+                  <b-icon-x></b-icon-x>
+                </b-button>
+              </template>
+            </b-input-group>
+          </b-col>
+        </b-row>
+
+        <template slot='invalid-feedback'>
+          <div v-html="fieldError('logo')"></div>
+          <div v-html="fieldError('logo_file')"></div>
+        </template>
+      </b-form-group>
+
+      <!--Room limit settings-->
+      <b-form-group
+        label-for="application-room-limit-input"
+        :description="$t('settings.application.roomLimit.description')"
+        :state='fieldState("room_limit")'
+      >
+
+        <template v-slot:label>
+          {{ $t('settings.application.roomLimit.title') }}
```

Comment:
Why we need the extra template? The label could be set in the form group through an attribute!